scout/server/blueprints/panels/controllers.py

```diff
@@ -265,3 +265,10 @@ def panel_export(store, panel_obj):
     panel_obj["name_and_version"] = full_name
 
     return dict(panel=panel_obj)
+
+
+def panels_to_json(store, panel_name):
```

Comment:
This function is named panels_to_json but it's not really what it does. The json conversion happens in the view instead. Perhaps you don't even need it..

Reply:
It's the same as the other API calls, but I do agree!